pythonforandroid/bootstraps/common/build/templates/Service.tmpl.java

```diff
@@ -11,19 +11,34 @@
 import org.kivy.android.PythonService;
 import org.kivy.android.PythonActivity;
 
+import android.content.pm.PackageManager;
+import android.support.v4.app.ActivityCompat;
+import android.Manifest;
+import android.app.NotificationChannel;
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.content.Context;
+import android.support.v4.app.NotificationCompat;
+import android.graphics.Color;
+import android.app.Service;
+import android.content.ComponentName;
+import android.app.PendingIntent;
 
 public class Service{{ name|capitalize }} extends PythonService {
+
+    private boolean autoRestartService = true;
+
     {% if sticky %}
     @Override
     public int startType() {
-        return START_STICKY;
+        return START_NOT_STICKY;
```

Comment:
I'm really not getting that part :thinking: we have the `if sticky` in the template, but we decide to return not sticky.
There's something fishy about it no?